docs/proposal-config-changes.md

```diff
@@ -392,25 +396,49 @@ Matches global config change to `[task events]mail retry delays`.
 See
 [discussion](https://github.com/cylc/cylc-flow/issues/3404#issuecomment-540190164).
 
-### `[scheduler][events]abort if any task fails` -> `[scheduler][events]abort if any unexpected task fails`
+### `[scheduler][events]abort if any task fails`  -> obsolete
 
-See change above (different name proposed).
+Make command line option instead (discussed at CylcCon).
 
-### `[cylc][environment]` -> `[scheduler][environment]`
+### `[cylc][environment]` -> obsolete
 
-Makes sense to move this.
-
-Rename to `handler environment`?
-(think this was proposed but can't find discussion)
+Discussed at CylcCon.
 
 ### `[cylc]health check interval` -> `[scheduler]health check interval`
 
 Matches global config change.
 
+### `[cylc]` time settings -> `[scheduler]`
+
+This covers `UTC mode`, `cycle point format`,
+`cycle point num expanded year digits` and `cycle point time zone` (discussed at
+CylcCon).
+
+### `[cylc][simulation]disable suite event handlers` -> ???
+
+Agreed at CylcCon to move this and make it positive.
+Notes suggest `[scheduling]use suite event handlers`.
+However current setting only applies to simulation and dummy mode (and defaults to true) -
+we don't need this setting for real workflows do we (and we want these handles disabled by default in simulation and dummy mode)?.
+How about `[scheduler][events]enable simulation handlers`?
```

Comment:
Not sure about this bit

Reply:
Shall we just remove it and automatically disable event handlers in dummy and sim modes?  It's easy to test event handlers, if you need to, in a real test suite written for that purpose.  I can't really think of a good use case for this when running a real suite in dummy or sim mode.

Reply:
Agreed - I've updated the proposal to make this setting obsolete